### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by aryehlev**
all cahnges from pull request login- will make sure to do properly separate next time

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\97254\\AppData\\Local\\Microsoft\\WindowsApps\\python.exe"
+}
```
Comment (simzacks) [inline .vscode/settings.json]: this file is still here. Maybe you need to remove it from the commit.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .vagrant/
 .idea/
 *.pyc
+.vscode/
```
Comment (simzacks) [inline .gitignore]: end files with a new line

File: superx/routes/home.py [hunk 3]
```diff
@@ -8,6 +8,8 @@ def home():
     return render_template('home.html', basket=basket)
 
 
-def login():
-    return render_template('login.html')
+def logged_in():
+   
+    basket = db.session.query(Basket).get(1)
 
+    return render_template('home-signedin.jinja2', basket=basket)
```
Comment (simzacks) [inline superx/routes/home.py]: end file with a new line

File: superx/routing.py [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
-    return home.login()
+    return signup.login()
+
+@app.route('/register', methods=['GET', 'POST'])
+def register():
+    return signup.register()
+
+@app.route('/inhome', methods=['GET', 'POST'])
+def logged_in():
+   return home.logged_in()
```
Comment (simzacks) [inline superx/routing.py]: end file with a new line

File: superx/routes/signup.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+def register():
+    form = RegisterForm()
+
+    if form.validate_on_submit():
+        hash_password = generate_password_hash(form.password.data, method='sha256')
+        new_user = User(name=form.username.data, email=form.email.data, password=hash_password) 
+        db.session.add(new_user)
+        db.session.commit()
+        # return '<h1>' + form.username.data + ' ' + form.password.data + ' ' +  form.email.data + '</h1>'
+
```
Comment (simzacks) [inline superx/routes/signup.py]: remove the unnecessary comment

File: .github/workflows/pylint.yml [hunk 6]
```diff
@@ -0,0 +1,27 @@
+name: Pylint
```
Comment (simzacks) [inline .github/workflows/pylint.yml]: This file should be submitted as a separate PR
https://stackoverflow.com/questions/8450036/how-to-open-multiple-pull-requests-on-github

**Review by simzacks — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (aryehlev) [reply .vscode/settings.json]: fixed it, its a bug in vscode

File: .gitignore (hunk 2 above)
Comment (aryehlev) [reply .gitignore:4]: i have changed these files back to old format of routes dir and routes.py

File: superx/routes/signup.py (hunk 5 above)
Comment (aryehlev) [reply superx/routes/signup.py]: ok

File: .github/workflows/pylint.yml (hunk 6 above)
Comment (aryehlev) [reply .github/workflows/pylint.yml]: ok i will do this, but i will wait till this PR is approved because i have done a mistake and dont want balagan.
i have another 2 separtae PRs that i will submit after approvel. thanks!

**Review by simzacks — APPROVED** [review]
